.github/workflows/cron-job-its.yml

```diff
@@ -60,8 +61,8 @@ jobs:
     uses: ./.github/workflows/reusable-standard-its.yml
     needs: build
     with:
-      build_jdk: 8
-      runtime_jdk: 11
+      build_jdk: 21.0.4
```

Comment:
This should be changed. These experiments can be done on private branches and not on the main branch. 
Build jdk <=runtimeJdk.